vendor/github.com/containers/storage/pkg/archive/changes.go

```diff
@@ -116,6 +116,16 @@ func aufsWhiteoutPresent(root, path string) (bool, error) {
 	return false, err
 }
 
+func aufsRedirectDir(root, path string, fi os.FileInfo) ([]string, error) {
+	var redirectPaths []string
+	redirect, err := system.Lgetxattr(filepath.Join(root, path), GetOverlayXattrName("redirect"))
+	if err != nil {
+		return redirectPaths, err
+	}
+	fmt.Println("----not sure what to do here redirect-----:", string(redirect))
```

Comment:
I am not sure what needs to be done here and how this is different from the function definition in https://github.com/containers/podman/pull/18690/files#diff-7ed9846e3d8281986b5773425a048cdcade7a60ab7fdef5a02ede7b6bd55442bR343.

Reply:
once you get the redirect xattr, do you perform the lookup in the lower layers?  This is the tricky part, since this could go on for several layers and you need to emulate the overlay lookup (e.g. you must stop when you find an opaque directory)

Reply:
Yes, once I find a redirect dir I do the lookup in the lower layers, it is done in this function https://github.com/containers/podman/pull/18690/files#diff-7ed9846e3d8281986b5773425a048cdcade7a60ab7fdef5a02ede7b6bd55442bR343. Does the same thing need to happen in changes.go?